### PR TITLE
aug dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-august-2022.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+Several plugins have been improved this past month. The S3 Resource Plugins for the 4.3 series and 4.2 series now support Amazon Glacier / Deep Archive. Not only that, but several bugs have been fixed as well (e.g. support for `itouch` and `istream --append`).
+
+Work on the Indexing Capability Plugin has been merged. It now handles configuration properties that accept integers and strings more gracefully. It also handles `iput --metadata` without any issues. Work to support an SSL-capable elasticlient is in full swing too.
+
+Work to replace all use of the Jansson JSON library in the Audit AMQP Rule Engine Plugin has been going well. Aside from the plugin, the [ELK stack](https://github.com/irods/contrib/tree/main/irods_audit_elk_stack) has received some major updates. All components in the project have been updated to a supported version. The project now works with the iRODS 4.3 series and 4.2 series.
```
Comment (trel) [inline content/posts/irods-development-update-august-2022.md:24]: ‘used for training’

Comment (korydraughn) [reply content/posts/irods-development-update-august-2022.md:24]: Is that ...
```
The project used for training now works with the iRODS 4.3 series and 4.2 series.
```
or
```
The project now works with the iRODS 4.3 series and 4.2 series used for training.
```

Comment (trel) [reply content/posts/irods-development-update-august-2022.md]: …`, used for training, has received some major updates.`

Comment (korydraughn) [reply content/posts/irods-development-update-august-2022.md]: Will change.

Comment (korydraughn) [reply content/posts/irods-development-update-august-2022.md]: Done

File: content/posts/irods-development-update-august-2022.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
+Title: iRODS Development Update: August 2022
+Date: 2022-08-30 12:00
+Author: Kory Draughn
+Slug: irods-development-update-august-2022
+Status: published
+
+
+Hello everyone! August is on its way out and that means it's time for another development update.
+
+To start things off, TRiRODS was a success. Our interns did a wonderful job on their projects and presentations. If you're interested in hearing about their experience, you can view their talks online at https://irods.org/trirods/.
```
Comment (trel) [inline content/posts/irods-development-update-august-2022.md:10]: please hyperlink the URL...

`[https://irods.org/trirods](https://irods.org/trirods)`

File: content/posts/irods-development-update-august-2022.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
+Title: iRODS Development Update: August 2022
+Date: 2022-08-30 12:00
```
Comment (trel) [inline content/posts/irods-development-update-august-2022.md:2]: let's change to 15:00

**Comment by trel** [comment]
tweak then squash, will merge.

**Pull request merged**